tests/e2e-instrumentation/instrumentation-nginx-contnr-secctx/01-install-app.yaml

```diff
@@ -22,7 +22,6 @@ spec:
         securityContext:
           runAsUser: 1000
           runAsGroup: 3000
-          fsGroup: 3000
```

Comment:
Why is this marked as non valid? https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
This is a container security context, which doesn't have a `fsGroup` field: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#securitycontext-v1-core. As opposed to a Pod security context, which does have one: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#podsecuritycontext-v1-core